src/components/List/Row.js

```diff
@@ -21,41 +23,52 @@ const ListRow = ({
       expandMultiple,
       renderIntoPortal,
       ...restItemContainerProps
-    }) => (
-      <ListRowContent
-        rowItem={rowItem}
-        isOpen={determineIfOpen(expandMultiple, openIndex, index)}
-        index={index}
-        onOverflowClick={() => {
-          renderIntoPortal({ children, contentType: "mobile", data: rowItem });
+    }) => {
+      const params = {
+        rowItem,
+        isOpen: determineIfOpen(expandMultiple, openIndex, index),
+        index,
+        onOverflowClick: () => {
+          renderIntoPortal({
+            children,
+            contentType: "mobile",
+            data: rowItem
```

Comment:
Should you omit passing `rowItem` as the `data` property on this payload when the `isPure` prop equals true, given that it is not required for that variant?

Reply:
Agree, done.